test_enet.py

```diff
@@ -71,6 +71,24 @@ def test_connect(self):
         self.assertEquals(client_connected, True)
         self.assertEquals(server_connected, True)
 
+    def test_socketsend(self):
+
+        self.send_done = False
```

Comment:
It is needed, because send_done is defined in another scope. The easiest way to access a variable from another scope is to have a reference to it through an object. `self` is a perfect choice for such object.